src/sass/_placeholders.scss

```diff
@@ -0,0 +1,16 @@
+%time-hover-animation {
+  transition: $universal-transition-timing;
+  -webkit-transition: $universal-transition-timing;
+}
+
+%bg-primary-hover-animation {
```

Comment:
nie rozumiem tej i poniższych nazw. mają `animation` w sobie, tyczą się tylko kolorów, a ich użycie jest zagadkowe.

zamiast składać te animacje z 2-3 extendów zdefiniuj każdy z nich w sposób, który zawiera w sobie wszystkie potrzebne rzeczy
```
%bg-hover-animation { // albo inna nazwa
transition: $universal-transition-timing;
background-color: $primary;
}
```
dodatkowo nie wiąż ich z "primary" w tym wypadku, bo jeśli klient przyjdzie i powie "ja jednak chcę ten drugi kolor na hover", to skończysz z np.
```
%bg-primary-hover-animation {
background-color: $secondary-color;
}
```

Reply:
Zmiany zostały dodane do brancha *bug-wdp-190203-9*. 
Ograniczenie do trzech placeholderów w pliku.